Makefile

```diff
@@ -94,13 +94,13 @@ registry-server: cd pkg/oci/registry
 .PHONY: docker-registry-server
 docker-registry-server:
 	docker buildx build \
-	-t ${REG_IMG}:${TAG} \
+	-t ${REG_IMG}:${REG_TAG} \
```

Comment:
@souleb I renamed these in case your want a different TAG for reg, or you don't want any REG_TAG at all but you want TAG... :)